src/pudl/transform/ferc1.py

```diff
@@ -784,10 +785,7 @@ def process_dbf(self, raw_dbf: pd.DataFrame) -> pd.DataFrame:
             .pipe(self.assign_record_id, source_ferc1=SourceFerc1.DBF)
             .pipe(self.drop_unused_original_columns_dbf)
             .pipe(self.assign_utility_id_ferc1, source_ferc1=SourceFerc1.DBF)
-            .pipe(
```

Comment:
just a formatting change here -- got rid of the trailing comma since it fits on one line and that's how the other similar calls are.